deploy/template/etc/star_taskcluster-worker_net.crt

```diff
@@ -22,43 +51,18 @@ CCsGAQUFBwIBFhxodHRwczovL3d3dy5kaWdpY2VydC5jb20vQ1BTMAgGBmeBDAEC
-gBgNyK1d98RFdfvaNEKSZful3s8z1TANBgkqhkiG9w0BAQsFAAOCAQEAL1LIV/9u
-1Ytn2fhTdrzbNe1IKvGjT6XytP7/1bbIqDJksN6b5N2zapUoQ1rxRWzA1tSuImG+
-5QC8PTVDvtt53y3Hqccm22y7hgNjLLE8umhoJg02tJWrMocQLMBHtC+JolFOex5r
-rIz2wHE4BHBvZ0hCNJT2MiX0Vo9OuweaBHdGk9Hl6Oo96oZAIsDZzXQTqy+14aFl
-WOM3Wls4JRi6q/DXTu/3mALCtEryBXXase06EW9Mn6FxdYTwLH0dDldcR7/GWY1h
-DigawwPkyHpFgUpqwNBAfX73ap7/oy2NVtoYnE1Y5IV2ORAobGl1iWn3gXVyIBvz
-slJh+pGgYHPBPQ==
------END CERTIFICATE-----
------BEGIN CERTIFICATE-----
-MIIElDCCA3ygAwIBAgIQAf2j627KdciIQ4tyS8+8kTANBgkqhkiG9w0BAQsFADBh
-MQswCQYDVQQGEwJVUzEVMBMGA1UEChMMRGlnaUNlcnQgSW5jMRkwFwYDVQQLExB3
-d3cuZGlnaWNlcnQuY29tMSAwHgYDVQQDExdEaWdpQ2VydCBHbG9iYWwgUm9vdCBD
-QTAeFw0xMzAzMDgxMjAwMDBaFw0yMzAzMDgxMjAwMDBaME0xCzAJBgNVBAYTAlVT
-MRUwEwYDVQQKEwxEaWdpQ2VydCBJbmMxJzAlBgNVBAMTHkRpZ2lDZXJ0IFNIQTIg
-U2VjdXJlIFNlcnZlciBDQTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEB
-ANyuWJBNwcQwFZA1W248ghX1LFy949v/cUP6ZCWA1O4Yok3wZtAKc24RmDYXZK83
-nf36QYSvx6+M/hpzTc8zl5CilodTgyu5pnVILR1WN3vaMTIa16yrBvSqXUu3R0bd
-KpPDkC55gIDvEwRqFDu1m5K+wgdlTvza/P96rtxcflUxDOg5B6TXvi/TC2rSsd9f
-/ld0Uzs1gN2ujkSYs58O09rg1/RrKatEp0tYhG2SS4HD2nOLEpdIkARFdRrdNzGX
-kujNVA075ME/OV4uuPNcfhCOhkEAjUVmR7ChZc6gqikJTvOX6+guqw9ypzAO+sf0
-/RR3w6RbKFfCs/mC/bdFWJsCAwEAAaOCAVowggFWMBIGA1UdEwEB/wQIMAYBAf8C
-AQAwDgYDVR0PAQH/BAQDAgGGMDQGCCsGAQUFBwEBBCgwJjAkBggrBgEFBQcwAYYY
-aHR0cDovL29jc3AuZGlnaWNlcnQuY29tMHsGA1UdHwR0MHIwN6A1oDOGMWh0dHA6
-Ly9jcmwzLmRpZ2ljZXJ0LmNvbS9EaWdpQ2VydEdsb2JhbFJvb3RDQS5jcmwwN6A1
-oDOGMWh0dHA6Ly9jcmw0LmRpZ2ljZXJ0LmNvbS9EaWdpQ2VydEdsb2JhbFJvb3RD
-QS5jcmwwPQYDVR0gBDYwNDAyBgRVHSAAMCowKAYIKwYBBQUHAgEWHGh0dHBzOi8v
-d3d3LmRpZ2ljZXJ0LmNvbS9DUFMwHQYDVR0OBBYEFA+AYRyCMWHVLyjnjUY4tCzh
-xtniMB8GA1UdIwQYMBaAFAPeUDVW0Uy7ZvCj4hsbw5eyPdFVMA0GCSqGSIb3DQEB
-CwUAA4IBAQAjPt9L0jFCpbZ+QlwaRMxp0Wi0XUvgBCFsS+JtzLHgl4+mUwnNqipl
-5TlPHoOlblyYoiQm5vuh7ZPHLgLGTUq/sELfeNqzqPlt/yGFUzZgTHbO7Djc1lGA
-8MXW5dRNJ2Srm8c+cftIl7gzbckTB+6WohsYFfZcTEDts8Ls/3HB40f/1LkAtDdC
-2iDJ6m6K7hQGrn2iWZiIqBtvLfTyyRRfJs8sjX7tN8Cp1Tm5gr8ZDOo0rwAhaPit
-c+LJMto4JQtV05od8GiG7S5BNO98pVAdvzr508EIDObtHopYJeS4d60tbvVS3bR0
-j6tJLp07kzQoH3jOlOrHvdPJbRzeXDLz
+fQYKKwYBBAHWeQIEAgSCAW0EggFpAWcAdgC72d+8H4pxtZOUI5eqkntHOFeVCqtS
+6BqQlmQ2jh7RhQAAAXCirIw8AAAEAwBHMEUCIQCQEKE706XaIUyxsVwKGogMx/Dq
+oieeupKsAnc0ErOb5gIgfxrKD8P5JxmHz/BDdzSCkEuf/n+ibrWyoFzMdwXvRyEA
+dQBc3EOS/uarRUSxXprUVuYQN/vV+kfcoXOUsl7m9scOygAAAXCirIxqAAAEAwBG
+MEQCIANTxyqaqIV9/7wgWuHMfplSpbccV7XBulkW/cXlOijNAiBKm9X3NiKFXonO
+PvOzGYjgeTjJKOg4sForsUHLzmtB/wB2AO7Ale6NcmQPkuPDuRvHEqNpagl7S2oa
+FDjmR7LL7cX5AAABcKKsjHwAAAQDAEcwRQIhAKsTRCSt3wxysNAVqI3DCexiX42v
+5J5bYDpfj1EzhH1jAiB5N71llLJq+DLH4D3BzHKksSxSwUdsaaRp6fcXjKvp+zAN
+BgkqhkiG9w0BAQsFAAOCAQEAahviZAYWoCA6/U3tvmNVh8d3hM/Z154gvBEgFKCE
+2jl+S/QCnUxNcBSb5RLVmGMU+M0f52MSH6mg5S+g33txtVxKjMNrFydEenc2lKFA
+imp2UG3Ngj5j19GFU52p/7SRgEKlgI0HEJi+p9xJJMohB4v67CmM5xet1Px5n++t
+ipqxp5Fv1hutpFKrpK1zOgCDdyfzwAez531heI21SW6D4VFga3BpeTWn8FwVIjHZ
+UZThWKWFj/OC6h+4qEpRnaXulWrTA3bFD+xTFyKmI7rtWg+gAzN8PmzBf1oETEP/
+AhI7BJ4N1FI1YGKlh6wc7SqoVmoi7lPhMF3+Oo4w94DaYg==
```

Comment:
It looks like this omits the DigiCert intermediate certificate that used to be in this file.  That cert is important to clients that do not already have it in their local cache (e.g., curl, chrome).  It's the one starting on line 38 of this file in master.